lib/pure/includes/asyncfutures.nim

```diff
@@ -16,6 +16,10 @@ type
 
   FutureVar*[T] = distinct Future[T]
 
+  FutureStream*[T] = ref object of FutureBase   ## Special future that acts as
```

Comment:
Mark this as experimental in the docs, the API might change but yet we need it for a better httpclient.